common/components/chrome/SocialFooter.jsx

```diff
@@ -5,6 +5,9 @@ import NewsletterSignup from '../common/integrations/NewsletterSignup.jsx';
 import url from "../utils/url.js";
 import Section from "../enums/Section.js";
 
+const sectionsToHideFooter: $ReadOnlyArray<string> = [
```

Comment:
I think it's good we have the option to hide the footer but now that the live event page should take up vertical space beyond full viewport height, do we want to keep it removed? That is - was removing it a short-term "we need the space right now" or a long-term "We don't want that footer there at all" decision?